docs/ops/sort/NonMaxSuppression_5.md

```diff
@@ -0,0 +1,121 @@
+## NonMaxSuppression<a name="NonMaxSuppression"></a> {#openvino_docs_ops_sort_NonMaxSuppression_5}
+
+**Versioned name**: *NonMaxSuppression-5*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: *NonMaxSuppression* performs non maximum suppression of the boxes with predicted scores.
+
+**Detailed description**: *NonMaxSuppression* performs non maximum suppression algorithm as described below:
+
+1.  Let `B = [b_0,...,b_n]` be the list of initial detection boxes, `S = [s_0,...,s_N]` be  the list of corresponding scores.
+2.  Let `D = []` be an initial collection of resulting boxes.
+3.  If `B` is empty then go to step 8.
+4.  Take the box with highest score. Suppose that it is the box `b` with the score `s`.
+5.  Delete `b` from `B`.
+6.  If the score `s` is greater or equal than `score_threshold`  then add `b` to `D` else go to step 8.
+7.  For each input box `b_i` from `B` and the corresponding score `s_i`, set `s_i = s_i * func(IOU(b_i, b))` and go to step 3.
+8.  Return `D`, a collection of the corresponding scores `S`, and the number of elements in `D`.
+
+Here `func(iou) = iou if iou <= iou_threshold else 0` when `soft_nms_sigma == 0`, else `func(iou) = exp(-0.5 * iou * iou / soft_nms_sigma) if iou <= iou_threshold else 0`.
```

Comment:
According to the original paper on Soft-NMS (https://arxiv.org/pdf/1704.04503.pdf) for the cases when `soft_nms_sigma == 0` we should leave the score unchanged if `iou <= iou_threshold` thus `func(iou)` should be equal to `1` in this case. And for cases when `soft_nms_sigma != 0` it seems like this we should calculate `func = exp(iou * iou / soft_nms_sigma)` for all cases no matter if `iou` is less than threshold or not. 
Am I correct?

Reply:
Fixed.

Reply:
Great catch!